tracing-subscriber/src/fmt/env_logger_compat.rs

```diff
@@ -0,0 +1,57 @@
+use crate::{EnvFilter, FmtSubscriber};
+use std::{error::Error, fmt};
+use tracing::subscriber::{self, SetGlobalDefaultError};
+
+#[cfg(feature = "tracing_log")]
+use tracing_log::{LogTracer, SetLoggerError};
+
+#[derive(Debug)]
+enum InitError {
+    #[cfg(feature = "tracing_log")]
+    SetLogger(SetLoggerError),
+    SetGlobalDefault(SetGlobalDefaultError),
+}
+
+impl fmt::Display for InitError {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> Result<(), fmt::Error> {
+        match self {
+            #[cfg(feature = "tracing_log")]
+            InitError::SetLogger(ref err) => err.fmt(f),
+            InitError::SetGlobalDefault(ref err) => err.fmt(f),
+        }
+    }
+}
+
+impl Error for InitError {}
+
+/// Install a global tracing subscriber that listens for events and
+/// filters based on environment variables, if one is not already set.
+///
+/// Returns whether the initialization was successful.
+///
+/// If the `tracing_log` feature is enabled, this will also install
+/// the LogTracer to convert `Log` records into `tracing` `Event`s.
+pub fn try_init() -> Result<(), impl Error> {
+    #[cfg(feature = "tracing_log")]
+    tracing_log::LogTracer::init().map_err(InitError::SetLogger)?;
+
+    subscriber::set_global_default(
```

Comment:
why not just use `tracing_core::dispatcher::set_global_default`? that way, we don't need a `tracing` dependency.